site/en/blog/new-in-extensions-1/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: 'layouts/blog-post.njk'
+title: An example blog post
```

Comment:
missing title

Reply:
Good catch.

Reply:
Still looking for the preview. Does something need to happen to kick Netlify back to life?

Reply:
Netlify is reporting that it's deployed at the last commit, so you should be good?